CHANGELOG.md

```diff
@@ -1,3 +1,88 @@
+<a name="4.5.0"></a>
+## [4.5.0](https://github.com/osquery/osquery/releases/tag/4.5.0)
+
+[Git Commits](https://github.com/osquery/osquery/compare/4.4.0...4.5.0)
+
+### New Features
+
+- Aarch64 beta support for Linux ([#6612](https://github.com/osquery/osquery/pull/6612))
```

Comment:
Consensus from slack was that we wanted to thank the contributors here.

Reply:
My preference is to have thanks in the blog post https://github.com/osquery/osquery-site/pull/183

Can do both though.